src/include/test/unit_tdb_matrix.cc

```diff
@@ -201,3 +202,66 @@ TEMPLATE_TEST_CASE("tdb_matrix: preload", "[tdb_matrix]", float, uint8_t) {
     }
   }
 }
+
+TEST_CASE("tdb_matrix: MatrixBase template parameter", "[tdb_matrix]") {
+  // Load data.
+  tiledb::Context ctx;
+  int offset = 13;
+
+  size_t Mrows = 200;
+  size_t Ncols = 500;
+
+  using T = float;
+  using I = size_t;
+  using LayoutPolicy = stdx::layout_left;
+  using IdsType = uint8_t;
+
+  std::string tmp_matrix_uri = "/tmp/tmp_tdb_matrix";
```

Comment:
Should we use a more random pattern for the test filenames? 
We can do this consistently in a followup PR.

Reply:
Sure, added a story for it: https://app.shortcut.com/tiledb-inc/story/41375